src/renderer/hooks/useStorage.ts

```diff
@@ -2,7 +2,7 @@ import { useState } from "react";
 import { createStorage } from "../utils";
 import { CreateObservableOptions } from "mobx/lib/api/observable";
 
-export function useStorage<T>(key: string, initialValue?: T, options?: CreateObservableOptions) {
+export function useStorage<T>(key: string, initialValue: T, options?: CreateObservableOptions) {
```

Comment:
This function doesn't appear to be used anywhere

Reply:
Ah good catch. I changed this is a preemptive measure.